packages/trpc/server/routers/viewer/slots/util.ts

```diff
@@ -363,31 +356,27 @@ async function _getAvailableSlots({ input, ctx }: GetScheduleOptions): Promise<I
       shouldServeCache,
     });
 
-  // If contact skipping, determine if there's availability within two weeks
-  if (contactOwnerEmail && aggregatedAvailability.length > 0) {
+  // Fairness and Contact Owner disqualification reasons need Availability within 2 weeks
+  // if not, we attempt to re-open.
+  if (fallbackHosts && fallbackHosts.length > 0) {
```

Comment:
@joeauyeung rewritten this check, now any time there's fallbackHosts found I run the 2-weeks check; if there's not enough availability within the qualified users we use the fallbackHosts instead.

Reply:
Are we sure we want that? 

If reschedule with same host is enabled, we would do that too then. I think we shouldn't make that change in this PR